railties/test/application/rake/dbs_test.rb

```diff
@@ -734,6 +734,25 @@ def db_test_load_structure
         end
       end
 
+      test "db:prepare works when database and schema file do not exist" do
```

Comment:
This spec would fail on `master`.

```
E

Error:
ApplicationTests::RakeTests::RakeDbsTest#test_db:prepare_works_when_database_and_schema_file_do_not_exist:
RuntimeError: rails command failed (1): bin/rails db:prepare 2>&1
/Users/david-stosik/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/rake-13.0.1/lib/rake/application.rb:680: warning: conflicting chdir during another chdir block                                                                                                                                                            
/Users/david-stosik/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/rake-13.0.1/lib/rake/application.rb:701: warning: conflicting chdir during another chdir block                                                                                                                                                            
Created database 'railties_test'
Database 'railties_test' already exists
/Users/david-stosik/dev/rails/tmp/d20200427-74545-s8q9eg/app/db/schema.rb doesn't exist yet. Run `bin/rails db:migrate` to create it, then try again. If you do not intend to use a database, you should instead alter /Users/david-stosik/dev/rails/tmp/d20200427-74545-s8q9eg/app/config/application.rb to limit the frameworks that will be loaded.

    test/application/rake/dbs_test.rb:745:in `block (2 levels) in <class:RakeDbsTest>'
    test/application/rake/dbs_test.rb:740:in `chdir'
    test/application/rake/dbs_test.rb:740:in `block in <class:RakeDbsTest>'
```

It should pass with this PR's changes.